smoke_tests/conftest.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+from __future__ import print_function
+
+from __future__ import absolute_import
+import os
+
+import pytest  # type: ignore
+import six
+import yaml
+
+from smoke_tests.tools.compat import Path
+from scalyr_agent import compat
+
+
+def pytest_addoption(parser):
+    parser.addoption("--config", action="store")
```

Comment:
I assume this parameter refers to the circle ci config file, or?

In any case, please document it using help string.